curriculum/challenges/english/25-front-end-development/workshop-merge-sort/6564a9fe51964c229d5b7f4c.md

```diff
@@ -0,0 +1,31 @@
+---
+id: 6564a9fe51964c229d5b7f4c
+title: Step 2
+challengeType: 20
+dashedName: step-2
+---
+
+# --description--
```

Comment:
I feel like this can be lumped into step 1 because campers will have a lot of practice with defining functions with parameters. So I think it is fine to move this to step 1 and get rid of step 2 altogether.